### PR TITLE
SSR-3217 Migrate docusaurus Adding v1.12.0

### PR DESCRIPTION
**Pull request by nmanhit**

### CODE REVIEW

File: docs/document/i18n/ja/docusaurus-plugin-content-docs/current/components/desktop/table.md [hunk 1]
```diff
@@ -19,12 +19,15 @@ import { TableComponent } from "@site/static/js/samples/desktop/table.js"
 
 使用できるプロパティの一覧です。プロパティを指定して値を更新することができます。
 
+#### Table
+
```
Comment (dong0) [inline docs/document/i18n/ja/docusaurus-plugin-content-docs/current/components/desktop/table.md:23]: It can be removed.
```suggestion
```

**Review by TomokoMiyake — APPROVED** [review]
LGTM!

**Comment by nmanhit** [comment]
> There are some changed files for v1.11.0. This is v1.12.0 PR, but is it ok?
Yes, It is ok Because the changes in each version are different.
If you make changes in version v1.11.0, it does not affect version v1.12.0 and v1.12.0 is correct

**Comment by TomokoMiyake** [comment]
> > There are some changed files for v1.11.0. This is v1.12.0 PR, but is it ok?
> 
> Yes, It is ok Because the changes in each version are different. If you make changes in version v1.11.0, it does not affect version v1.12.0 and v1.12.0 is correct

I mean that there are changes for v1.11.0 although this is PR for v1.12.0 like the file below.
I think it's correct if this PR includes only v1.12.0 related file changes.
Please double-check it.
https://github.com/kintone-labs/kintone-ui-component/pull/1399/files#diff-ae79561ef6b1271f915726e44b2b1d8311d63c20d4e7e64ebf79a075265a4b26

**Comment by nmanhit** [comment]
> I mean that there are changes for v1.11.0 although this is PR for v1.12.0 like the file below. I think it's correct if this PR includes only v1.12.0 related file changes. Please double-check it. https://github.com/kintone-labs/kintone-ui-component/pull/1399/files#diff-ae79561ef6b1271f915726e44b2b1d8311d63c20d4e7e64ebf79a075265a4b26

Because this v1.12.0 was branched from the v1.11.0 branch. When I was fixing in the v1.12.0 branch, I saw v1.11.0 branch also had some whitespace don't correct the same as v1.12.0, so I fixed it as well.
Is it ok? if not I will revert it in v1.11.0

**Comment by TomokoMiyake** [comment]
> > I mean that there are changes for v1.11.0 although this is PR for v1.12.0 like the file below. I think it's correct if this PR includes only v1.12.0 related file changes. Please double-check it. https://github.com/kintone-labs/kintone-ui-component/pull/1399/files#diff-ae79561ef6b1271f915726e44b2b1d8311d63c20d4e7e64ebf79a075265a4b26
> 
> Because this v1.12.0 was branched from the v1.11.0 branch. When I was fixing in the v1.12.0 branch, I saw v1.11.0 branch also had some whitespace don't correct the same as v1.12.0, so I fixed it as well. Is it ok? if not I will revert it in v1.11.0

I understood it now! And I think it's ok to include it in this PR.
On other PR, it will be best if it only includes the specific version changes.

**Pull request merged**